packages/dx-scheduler-core/src/plugins/appointment-form/utils.ts

```diff
@@ -67,3 +68,22 @@ export const getAvailableRecurrenceOptions: OptionsFormatterFn = getMessage =>
     text: getMessage(type),
     id: type,
   }));
+
+export const getCountDependingOnRecurrenceType = (frequency: number) => {
+  switch (frequency) {
+    case RRULE_REPEAT_TYPES.YEARLY:
+      return BASIC_YEALY_COUNT;
+    case RRULE_REPEAT_TYPES.MONTHLY:
+      return BASIC_MONTHLY_COUNT;
+    case RRULE_REPEAT_TYPES.WEEKLY:
+      return BASIC_WEEKLY_COUNT;
+    case RRULE_REPEAT_TYPES.DAILY:
+      return BASIC_DAILY_COUNT;
+    default:
+      return 1;
+  }
+};
+
+export const checkNumber: PureComputed<
+  [number], boolean
+> = number => number > 0 && number <= Number.MAX_SAFE_INTEGER;
```

Comment:
This function has called `checkNumber` but negative values are also numbers. I think we should change the name